### PR TITLE
Bugfix for wrong redirects

### LINKED ISSUE #784: Bug: wrong redirect when super admin confirms his rights in projectSettingsMenu

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR closes #784 

### What's done:
- [x] Fixed a bug with wrong redirect
- [x] Improved modals to have changeable button labels

### CODE REVIEW

**Review by petertrr — COMMENTED** [review]
What was the bug with redirect and how has it been fixed? It's not clear from code and there is no description in the issue

**Comment by sanyavertolet** [comment]
> What was the bug with redirect and how has it been fixed? It's not clear from code and there is no description in the issue

Yes, I used wrong enum element processing notification. Now there is a pure notification that is shown to super admin.
Now I use errorModalWindow because it has only one button.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt [hunk 1]
```diff
@@ -184,10 +190,10 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
     private fun showNotification(notificationLabel: String, notificationMessage: String) {
         setState {
-            confirmationType = ConfirmationType.GLOBAL_ROLE_CONFIRM
-            isConfirmWindowOpen = true
-            confirmLabel = notificationLabel
-            confirmMessage = notificationMessage
+            isErrorOpen = true
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt:193]: So basically you're now reusing an existing modal window not only for errors, but for confirmations too?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RBuilderUtils.kt [hunk 2]
```diff
@@ -32,13 +31,15 @@ enum class ConfirmationType {
  * @param isErrorOpen flag to handle error
  * @param errorLabel label of error
  * @param errorMessage message of error
+ * @param closeButtonLabel label that will be shown on the only button
  * @param handler handler to close
  * @return modal
  */
 fun RBuilder.runErrorModal(
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RBuilderUtils.kt:38]: These things can be refactored: converted to a `FC` with props containing all these settings, and renamed from `run...Modal` to something like `singleButtonModal` and `doubleButtonModal`. Can be done as a separate PR

**Review by petertrr — APPROVED** [review]

**Pull request merged**